src/Http/Controllers/GoogleAnalyticsController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tightenco\NovaGoogleAnalytics\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Arr;
+use Tightenco\NovaGoogleAnalytics\File;
+use Illuminate\Routing\Controller;
+use Spatie\Analytics\Analytics;
+use Spatie\Analytics\Period;
+
+class GoogleAnalyticsController extends Controller
+{
+    public function index(Request $request)
+    {
+        $limit = $request->input('limit', 10);
+        $page = $request->input('page', 1);
+        $offset = ($page - 1) * $limit;
+        $duration = $request->input('duration', 'week');
+        $searchTerm = $request->input('s', null);
+        $sortBy = $request->input('sortBy', 'ga:pageviews');
+        $sortDirection = $request->input('sortDirection', 'desc');
+        $filter = is_null($searchTerm) ? null : 'ga:pageTitle=@' . $searchTerm . ',ga:pagePath=@' . $searchTerm;
```

Comment:
What do you think about this?

```suggestion
        $filter = $searchTerm ? sprintf('ga:pageTitle=@%s,ga:pagePath=@%s', $searchTerm, $searchTerm) : null;
```

Reply:
Also, since this variable is only used once, we could put it inline on line 33